### ISSUE TITLE
loss value is nan and training gets stopped

### ISSUE THREAD

**Issue by spaul13** [Okery/YOLOv5-PyTorch#10]
I am getting nan as loss value thats why the training code gets stopped. can anyone please tell me how to resolve this issue. 

![image](https://user-images.githubusercontent.com/20827182/171697227-cf3da0de-5725-4606-ac3f-39486dd5ec45.png)

**Comment by salidw** [comment]
I found the same problem. Please suggest.
